lib/lightning/invocation/dataclip.ex

```diff
@@ -73,7 +73,11 @@ defmodule Lightning.Invocation.Dataclip do
   end
 
   defp remove_configuration(changeset) do
-    add_error(changeset, :body, "must be a map")
+    if changeset |> get_change(:body) |> is_nil() do
+      changeset
+    else
+      add_error(changeset, :body, "must be a map")
+    end
   end
```

Comment:
Can we just combine the two cases? I think it's less than obvious now given the "fall-through" function calls `get_change`.

This can be wrapped up in a `case changeset |> get_change(:body) do ...` and then checking the type returned from that.